Krypt/Source/KeyFingerprintPair.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  KeyFingerprintPair.swift
+//  Krypt
+//
+//  Created by Sun Bin Kim on 24.06.19.
+//
+
+import Foundation
+
+/// Holds key and fingerprint
+//  fingerprintFile is hex encoded and contains version in front
+public struct KeyFingerprintPair {
```

Comment:
This is `MedSticker` specific right?
```suggestion
public struct MedStickerKeyFingerprintPair {
```